### DISCUSSION TITLE
Use of variables inside a Field

### DISCUSSION THREAD

**Discussion by wiwa1978** [filamentphp/filament#12343] [Help]
### Package

Panel builder

### Package Version

v3.2.60

### How can we help you?

I want to display information inside a Field. I have used a `Text Input` below because there is no `Label` field. Ideally I just want to display the plan name into a Form using a label (but label is not documented in the list of [supported fields](https://filamentphp.com/docs/3.x/forms/fields/getting-started#available-fields.)

In any case, I would like to use the variable inside a field. Can this be done?
```
public function form(Form $form): Form {
    $plan = $organisation->getPlan();
    $plan_name= $plan->name; 

    return $form
        ->schema([
            TextInput::make($plan_name)  //this line
              ->label('Name')
          ]);
}

**Comment by leandrocfe** [comment]
Honestly, I didn't understand it. Do you mean, a placeholder to show the value?
https://filamentphp.com/docs/3.x/forms/layout/placeholder

**Comment by micraux** [comment]
Do you just want to display the plan name and don't let the user modify it?

If so, have a look at [Disabling a field](https://filamentphp.com/docs/3.x/forms/fields/getting-started#disabling-a-field).
